### DISCUSSION TITLE
Required support to implement URIHandler

### DISCUSSION THREAD

**Discussion by Lokeshagit** [teltek/gst-plugin-ndi#108] [Q&A] [answered]
Hello Team,

We have referred uri_handler.rs and tried to implement same uri_handler in our code.

error[E0433]: failed to resolve: could not find `element` in `super`
   --> src/ndisrc/imp.rs:679:34
    |
679 | pub trait URIHandlerImpl: super::element::ElementImpl {
    |                                  ^^^^^^^ could not find `element` in `super`

Please help us to resolve this issue and provide some code so we can refer and implement at our end.

Thanks,

**Comment by sdroege** [comment, marked as answer]
A newer version of the element can be found here: https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/tree/main/net/ndi

And you can find an example of the required code to implement the interface here: https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/blob/93051dba0ea344d7febc6142256f8fa46ae48ca4/net/reqwest/src/reqwesthttpsrc/imp.rs#L1260-1284

**Reply by Lokeshagit** [reply]
Hello Sebastian, 

Thanks for providing pointers and examples to solve our issue.
With the help of the guide, I've successfully added support for URI handlers.
